test/data/stages/zip/zip.mpp.yaml

```diff
@@ -0,0 +1,57 @@
+version: '2'
+sources:
+  org.osbuild.inline: {}
+pipelines:
+  - mpp-import-pipelines:
+      path: ../../manifests/fedora-vars.ipp.yaml
+  - mpp-import-pipeline:
+      path: ../../manifests/fedora-build-v2.ipp.yaml
+      id: build
+    runner:
+      mpp-format-string: org.osbuild.fedora{release}
+  - name: input-tree
+    build: name:build
+    stages:
+      - type: org.osbuild.copy
+        inputs:
+          inlinefile:
+            type: org.osbuild.files
+            origin: org.osbuild.source
+            mpp-embed:
+              id: '42'
+              text: '42
+
```

Comment:
Do we need it? `'`

Reply:
I'm not really sure why the text is formatted this way either, but this is also something that I referenced from `tar.mpp.yaml`: https://github.com/osbuild/osbuild/blob/main/test/data/stages/tar/tar.mpp.yaml#L21-L24. Do you think I should modify it to just be `'42'` or keep it as is to be consistent with the `tar.mpp.yaml`?

Reply:
I would keep it consistent. I guess they wanted to embed a few newlines in the file.